src/Tiqr/Legacy/TiqrUser.php

```diff
@@ -33,28 +36,16 @@
 class TiqrUser implements TiqrUserInterface
 {
     /**
-     * @var Tiqr_UserStorage_Interface
-     */
-    private $userStorage;
-
-    /**
-     * @var Tiqr_UserSecretStorage_Interface
+     * @param string $userId
      */
-    private $userSecretStorage;
-
-    /**
-     * @var string The userId
-     */
-    private $userId;
-
     public function __construct(
-        Tiqr_UserStorage_Interface $userStorage,
-        Tiqr_UserSecretStorage_Interface $userSecretStorage,
-        $userId
+        private readonly Tiqr_UserStorage_Interface $userStorage,
+        private readonly Tiqr_UserSecretStorage_Interface $userSecretStorage,
```

Comment:
Naming is hard, is it a secret storage or a storage for secrets?

Reply:
This storage interface is specifically for storing UserSecrets (which is a tiqr UL thing).

Reply:
So the var could be `$storageForUserSecrets` them

Reply:
Indeed it could. Leaving it like this for now.